coala_quickstart/generation/FileGlobs.py

```diff
@@ -28,6 +30,12 @@ def get_project_files(log_printer, printer, project_dir):
                       color="green")
         ignore_globs = get_gitignore_glob(project_dir)
 
+    if noninteractive:
+        if ignore_globs is None:
+            file = open(".gitignore", "w")
+            file.write("src")
+            ignore_globs = get_gitignore_glob(project_dir)
+
     if ignore_globs is None:
```

Comment:
@adtac I think also it's ok like this
`if ignore_globs is None:
    if non_interactive:
        ignore_globs = [ ]` 
Like you said, we don't need to create a file. And when we check
`
    if ignore_globs is None:
       ...
            "Which files do you want coala to ignore?",
`
It won't ask the question :smile: